src/components/Header/index.tsx

```diff
@@ -33,40 +36,52 @@ export function Header({ theme, toggleTheme }: Props) {
 
     return (
         <header>
+            <>
+                <div className="d-flex justify-content-center align-items-center w-100 mb-2 mt-4">
+                    <a
+                        className="text-decoration-none fw-bolder link-body-emphasis"
+                        href="/"
+                    >
+                        <h1>{SITE_NAME}</h1>
```

Comment:
I said this in a previous PR but it seems I need to keep telling same things. It is not correct to use `h1` tag as a brand name